cpp/ql/test/query-tests/Likely Bugs/Format/WrongNumberOfFormatArguments/a.c

```diff
@@ -15,4 +19,7 @@ void test_custom_printf1()
   myMultiplyDefinedPrintf2("%i", 0); // GOOD (we can't tell which definition is correct so we have to assume this is OK)
   myMultiplyDefinedPrintf2("%i", 0, 1); // GOOD (we can't tell which definition is correct so we have to assume this is OK)
   myMultiplyDefinedPrintf2("%i", 0, 1, 2); // BAD (too many format arguments regardless of which definition is correct) [NOT DETECTED]
+  myMultiplyDefinedPrintf3("%s", "%s"); // GOOD (we can't tell which definition is correct so we have to assume this is OK)
+  myMultiplyDefinedPrintf3("%s", "%s", "%s"); // GOOD (we can't tell which definition is correct so we have to assume this is OK)
+  myMultiplyDefinedPrintf3("%s", "%s", "%s", "%s"); // BAD (too many format arguments regardless of which definition is correct) [NOT DETECTED]
```

Comment:
Incredibly, it doesn't seem to be a compile error to pass `"%s"` to an argument of type `int`. But can you change `extraArg` to have type `const char *` in any case?

Reply:
Whoops, yes.  Fixed.